vercel.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": 2,
+    "builds": [
+        {
+            "src": "package.json",
+            "use": "@now/static-build",
+            "config": {
+                "distDir": "dist"
+            }
+        }
+    ],
+    "routes": [
+        { "src": "/", "dest": "/docs/index.html" },
+        { "src": "/(.*)", "dest": "/docs/$1" }

```

Comment:
Do you really need both? Ideally the first one should be enough but if it's not then the second one should suffice because it's a superset. Am I missing anything?

Reply:
With only the first one it was breaking when changing any route (when going to a specific class documentation). It was also breaking when importing the scripts of the theme.